### PR TITLE
Update scoop installation instructions

### PR DESCRIPTION
**Pull request by sitiom**
Point to the official buckets (https://github.com/spicetify/spicetify-cli/issues/1290#issuecomment-1062704879)

### CODE REVIEW

File: docs/getting-started/advanced-installation.md [hunk 1]
```diff
@@ -17,13 +17,30 @@ If you have problem with downloading, use this:
...
+
 ### Scoop
 
-Follow this guide: https://github.com/zwxi/Scoop-Spotify
+```powershell
+scoop install spicetify-cli
+```
 
-### Chocolatey
+### Note for Windows users
```
Comment (afonsojramos) [inline docs/getting-started/advanced-installation.md:30]: I'd remove this as we are already in the "windows" section.

File: docs/getting-started/advanced-installation.md [hunk 2]
```diff
@@ -17,13 +17,30 @@ If you have problem with downloading, use this:
...
+- To find the location of your Spotify installation, run `scoop prefix spotify`.
+
+```console
+$ scoop prefix spotify
+C:\Users\<username>\scoop\apps\spotify\current
+```
+
+After you have located it, set `spotify_path` to that directory in the config file:
+
+![scoop-spotify-path](https://user-images.githubusercontent.com/56180050/157407968-9d55a75a-bcd5-49ec-bbd1-5f315bd3c303.png)
```
Comment (afonsojramos) [inline docs/getting-started/advanced-installation.md:43]: I would remove this as it does not add much information, and the line numbering is somewhat confusing 😕

File: docs/getting-started/advanced-installation.md [hunk 3]
```diff
@@ -17,13 +17,30 @@ If you have problem with downloading, use this:
...
+#### Spotify installed from Scoop
+
+- To find the location of your Spotify installation, run `scoop prefix spotify`.
+
+```console
+$ scoop prefix spotify
+C:\Users\<username>\scoop\apps\spotify\current
+```
+
+After you have located it, set `spotify_path` to that directory in the config file:
```
Comment (afonsojramos) [inline docs/getting-started/advanced-installation.md:41]: * [...] in Spicetify's config file:

**Review by afonsojramos — CHANGES_REQUESTED** [review]
Looks good, just apply the suggested changes and I'll merge!

File: docs/getting-started/advanced-installation.md (hunk 2 above)
Comment (sitiom) [reply docs/getting-started/advanced-installation.md:43]: I was being consistent with the Flatpak Linux installation, but I guess I can fix the line numbering.

File: docs/getting-started/advanced-installation.md (hunk 1 above)
Comment (sitiom) [reply docs/getting-started/advanced-installation.md:30]: I was being consistent with the `Note for Linux users` section below but seems like that is under the `Linux and MacOS` section. I'll rename this to `Notes`.

Comment (sitiom) [reply docs/getting-started/advanced-installation.md]: I was being consistent with the `Note for Linux users` section below but seems like that is under the `Linux and MacOS` section. I'll rename this to `Notes`.

File: docs/getting-started/advanced-installation.md (hunk 2 above)
Comment (afonsojramos) [reply docs/getting-started/advanced-installation.md]: What is already there isn't the best example 😅 Thanks!

File: docs/getting-started/advanced-installation.md (hunk 1 above)
Comment (afonsojramos) [reply docs/getting-started/advanced-installation.md]: I like it, thanks!

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**